react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clickId: '000',
+      shown: false
+    };
```

Comment:
If you change `clickId` to `null`, then you don't need `shown`. When the user clicks on the same topic that is already open, set `clickId` back to `null`.